api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -276,17 +276,12 @@ message ExternalProcessor {
   // The default value is 5000 milliseconds (5 seconds) if not specified.
   google.protobuf.Duration deferred_close_timeout = 19;
 
-  // [#not-implemented-hide:]
   // Send body to the side stream server once it arrives without waiting for the header response from that server.
   // It only works for STREAMED body processing mode. For any other body processing modes, it is ignored.
   //
   // The server has two options upon receiving a header request:
   // 1. Instant Response: Send the header response as soon as the header request is received.
   // 2. Delayed Response: Wait for the body before sending any response.
-  //   If the server chooses the second option, it has two further choices:
```

Comment:
Thanks. This API change LGTM.

I am glad that we are moving in this direction. It is nature way for the existing 1:1 streaming mode.

Reply:
Agreed